Yousei.Connectors/Http/HttpConnector.cs

```diff
@@ -16,7 +16,7 @@ public HttpConnector() : base("http")
             DefaultConnection = new HttpConnection(default);
         }
 
-        protected override IConnection CreateConnection(Config configuration)
+        protected override IConnection? CreateConnection(Config configuration)
```

Comment:
Shouldn't the configuration also be nullable?

Reply:
No as it can never be `null` due to `SimpleConnector<T>` checking this before invoking `CreateConnection`.